plugins/katello/3.15/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,12 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important note:
+
+* Pulpcore installs a separate PostgreSQL database. The installer lacks support for external databases. This support is introduced in Katello 3.16.
+
+* Foreman ([since 2.0])(https://theforeman.org/manuals/2.0/index.html#Headlinefeatures) and Pulpcore requires PostgresSQL 10+. Databases should be upgraded to avoid conflicts. It is recommended to update to version 12.
```

Comment:
Not a native speaker, but isn't it plural?
```suggestion
* Foreman ([since 2.0])(https://theforeman.org/manuals/2.0/index.html#Headlinefeatures) and Pulpcore require PostgresSQL 10+. Databases should be upgraded to avoid conflicts. It is recommended to update to version 12.
```

Reply:
You are correct!